### ISSUE TITLE
Compatibility issue with newer Django (1.8/1.10)

### ISSUE THREAD

**Issue by MartinBorg** [toastdriven/restless#79]
`django.conf.urls.patterns()` is deprecated since Django 1.8 and removed in 1.10.
It is imported however in `dj.py`.
See https://docs.djangoproject.com/en/dev/releases/1.8/#django-conf-urls-patterns

**Comment by rougeth** [comment]
There's a PR that fix this https://github.com/toastdriven/restless/pull/75.

**Comment by MartinBorg** [comment]
Thanks, this solves the problem! Note, that the version compare seems to be wrong, e.g. `1.10` is not greater than `1.7.0`. For me, it worked with `StrictVersion()`, however:

    from distutils.version import StrictVersion
    from django import get_version as get_django_version
    if StrictVersion(get_django_version()) >= StrictVersion('1.7.0'):
        from django.conf.urls import url
    else:
        from django.conf.urls import patterns, url

**Comment by natsn** [comment]
When is this going to get resolved?

**Comment by jadsonbr** [comment]
Temporarily adapted the dj.py file on line 4 to make it as follows.

`from django.conf.urls include import, url`

and function in-line so that it was 74 in the following manner.

```
    @classmethod
    def urls(cls, name_prefix=None):
        """
        A convenience method for hooking up the URLs.

        This automatically adds a list & a detail endpoint to your URLconf.

        :param name_prefix: (Optional) A prefix for the URL's name (for
            resolving). The default is ``None``, which will autocreate a prefix
            based on the class name. Ex: ``BlogPostResource`` ->
            ``api_blogpost_list``
        :type name_prefix: string

        :returns: A ``patterns`` object for ``include(...)``
        """
        urlpatterns = [
            url(r'^$', cls.as_list(), name=cls.build_url_name('list', name_prefix)),
            url(r'^(?P<pk>\d+)/$', cls.as_detail(), name=cls.build_url_name('detail', name_prefix)),        
        ]
        return urlpatterns
```

**Comment by ElSaico** [comment]
Fixed by #84.
